chain/evm/compass.go

```diff
@@ -355,24 +356,32 @@ func (t compass) findLastValsetMessageID(ctx context.Context) (uint64, error) {
 }
 
 func (t compass) isArbitraryCallAlreadyExecuted(ctx context.Context, messageID uint64) (bool, error) {
+	blockNumber, err := t.evm.FindCurrentBlockNumber(ctx)
+
+	if err != nil {
+		return false, err
+	}
+	fromBlock := *big.NewInt(0)
+	fromBlock.Sub(blockNumber, big.NewInt(9999))
```

Comment:
Why 9999? Can you please add a comment?

Reply:
eth_getLogs and eth_newFilter are limited to a 10,000 blocks range so I set it 9999.